### PR TITLE
Ubuntu spacewalk-common-channels method

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9529

### CODE REVIEW

**Comment by Loquacity** [comment]
Waiting on response to this question: https://github.com/SUSE/spacewalk/issues/9529#issuecomment-553249603

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -132,7 +137,7 @@ ubuntu-1804-amd64-universe-uyuni ubuntu-1804-amd64-uyuni-client
 
 [IMPORTANT]
 ====
-You need all the new channels fully syncronized, including Universe (Universe contains important dependencies for salt), before bootstraping any Ubuntu client.
+You need all the new channels fully synchronized, including Universe (Universe contains important dependencies for salt), before bootstrapping any Ubuntu client.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc]:
```suggestion
You need all the new channels fully synchronized, including Universe (Universe contains important dependencies for Salt), before bootstrapping any Ubuntu client.
```

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 2]
```diff
@@ -104,11 +108,45 @@ When you have the channels set up, associate each channel with the appropriate r
...
+. At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
++
+----
+sudo zypper in spacewalk-utils
+----
+. Add the {ubuntu} channels.
+Adjust the version of the channel names to match your {ubuntu} version:
++
+----
+sudo spacewalk-common-channels ubuntu-1804-amd64
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: Are we sure this is the only channel? I'd say we are missing something else.

Please fire up a SUSE Manager 4.0 server and an Ubuntu client and give it a try :-)

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 3]
```diff
@@ -104,11 +108,45 @@ When you have the channels set up, associate each channel with the appropriate r
...
+----
+. Synchronize the new custom channels.
+
+[IMPORTANT]
+====
+You need all the new channels fully synchronized, including ``Universe``, before you can bootstrap an {ubuntu} client.
+The Universe channel contains important dependencies for Salt.
+====
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-ubuntu.adoc]: No, Universe is not needed for SUSE Manager. We provide the required packages from Universe at the client tools (but we don't do this for Uyuni)

File: modules/client-configuration/pages/clients-ubuntu.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-ubuntu.adoc]: Yeah, the original instructions had an asterisk, which I missed. Added them explicitly.

**Pull request merged**